runsc/container/container.go

```diff
@@ -939,6 +939,7 @@ func createGoferConf(overlayMedium config.OverlayMedium, mountType string, mount
 func (c *Container) initGoferConfs(ovlConf config.Overlay2, mountHints *boot.PodMountHints, rootfsHint *boot.RootfsHint) error {
 	// Handle root mount first.
 	overlayMedium := ovlConf.RootOverlayMedium()
+	overlaySize := ovlConf.RootOverlaySize()
 	mountType := boot.Bind
 	if rootfsHint != nil {
```

Comment:
For similar reasons as mentioned below, if rootfs is being configured by the rootfs annotations (rootfsHint != nil), then overlaySize should be reset.

Also, I think we should extend the rootfs annotation to include specifying size: https://github.com/google/gvisor/blob/d19c3bc9bcdf36fc592d634e9337dd4f84bec02d/runsc/boot/mount_hints.go#L230

Maybe `dev.gvisor.spec.rootfs.options` which has value like `size=...`. Later we can extend this to support more tmpfs options if need be.

Reply:
Added the option.

```
    "annotations": {
        "dev.gvisor.spec.rootfs.type": "bind",
        "dev.gvisor.spec.rootfs.source": "/",
        "dev.gvisor.spec.rootfs.overlay": "memory",
        "dev.gvisor.spec.rootfs.options": "size=1m"
    }
```